finalPrep/modZeroHero.js

```diff
@@ -2,40 +2,84 @@
 
 // Declare two variables - heroName AND specialAbility - set to strings
 
+var heroName = "Matt";
+var specialAbility = "Laser Eyes";
+
 // Declare two variables - greeting AND catchphrase
 //   greeting should be assigned to a string that uses concatenation to include the heroName
 //   catchphrase should be assigned to a string that uses interpolation to include the specialAbility
 
+var greeting = "Never fear, " + heroName + " is here!";
+console.log(greeting);
+
+var catchphrase = `Sorry guys, its ${specialAbility}!`
+console.log(catchphrase);
+
 // Declare two variables - power AND energy - set to integers
 
+var power = 9000;
+var energy = 75;
+
 // Declare two variables - fullPower AND fullEnergy
 //   fullPower should multiply your current power by 500
 //   fullEnergy should add 150 to your current energy
 
+var fullPower = power * 500;
+var fullEnergy = energy + 150;
+console.log(fullPower);
+console.log(fullEnergy);
+
 // Declare two variables - isHuman and identityConcealed - assigned to booleans
 
+var isHuman = true;
+var identityConcealed = false;
 
 // Declare two variables - archEnemies AND sidekicks
 //   archEnemies should be an array of at least 3 different enemy strings
 //   sidekicks should be an array of at least 3 different sidekick strings
 
+var archEnemies = ["Blob Man", "Blob Woman", "Blob Dog"];
+var sidekicks = ["Jelly Boy", "Old Boy", "Boy Boy"];
+
 // Print the first sidekick to your console
 
+console.log(sidekicks[0]);
+
 // Print the last archEnemy to the console
```

Comment:
How could you modify your code for this prompt so that no matter the length of the array, your code logs the last archEnemy? (This is a challenge question I throw at you because you're in a good place)